mypy/typeshed/stdlib/json/__init__.pyi

```diff
@@ -57,5 +70,6 @@ def load(
     parse_constant: Callable[[str], Any] | None = None,
     object_pairs_hook: Callable[[list[tuple[Any, Any]]], Any] | None = None,
     **kwds: Any,
-) -> Any: ...
+) -> object: ...
```

Comment:
Returning `Any` here is justified IMHO; otherwise you effectively require everyone to explicitly `cast` the result.

... or maybe it's possible to make the `JSONDecoder` generic in such a way that it can be used as return type here 🤔?

Reply:
fuck `Any`

Reply:
💪🏻